ui/admin/app/components/form/credential-store/index.hbs

```diff
@@ -2,8 +2,10 @@
   {{component
     (concat 'form/credential-store/' @model.type)
     model=@model
+    types=this.types
     submit=@submit
     cancel=@cancel
     changeType=@changeType
+    icon=this.icon
```

Comment:
Same here.  This duplicates information inferable from `@model.type` plus the model types hash.  The component could compute this via a getter.

Reply:
+1 to Randy's comment